src/pages/SignIn/SignIn.tsx

```diff
@@ -3,33 +3,38 @@ import ForgotPasswordModal from "./ForgotPasswordModal";
 import LoginModal from "./LoginModal";
 import SignUpModal from "./SignUpModal";
 
-export default function SignIn() {
+export default function SignIn({ signUp }: { signUp: boolean }) {
   const openLogin = () => {
-    //@ts-ignore
     window.HSOverlay.open(document.querySelector("#login-modal"));
   };
 
   const openSignUp = () => {
-    // @ts-ignore
     window.HSOverlay.open(document.querySelector("#sign-up-modal"));
   };
 
   const openForgotPassword = () => {
-    // @ts-ignore
     window.HSOverlay.open(document.querySelector("#forgot-pass-modal"));
   };
 
   return (
-    <div className="m-8">
-      <Button
-        label="Get Started"
-        associatedModalId="sign-in-modal"
-        onClick={openSignUp}
-      />
-      <ForgotPasswordModal
-        id="forgot-pass-modal"
-        onLoginClick={openLogin}
-      />
+    <div>
+      {signUp ? (
+        <Button
+          label="Get Started"
```

Comment:
this button should have a text size of 21 according to design (it is 14 rn). You can take in a className prop to the button component to support custom styles.